test/utils/uniswapData.ts

```diff
@@ -116,9 +116,12 @@ export async function getPool(tokenA: Token, tokenB: Token, feeAmount: FeeAmount
 
 // use some sane defaults
 export function swapOptions(options: Partial<SwapOptions>): SwapOptions {
+  // If theres a fee this counts as "slippage" for the amount out, so take it into account
+  let slippageTolerance = new Percent(5, 100)
+  if (!!options.fee) slippageTolerance = slippageTolerance.add(options.fee.fee)
```

Comment:
so basically..it's out of the scope of the sdk whether the fee is included in the slippageTolerance or no?t.... I think thats good..

Reply:
the sdk counts fee as part of the slippage tolerance. So integrators need to combine them